radio/src/logs.cpp

```diff
@@ -353,6 +357,10 @@ void logsWrite()
         }
       }
       f_printf(&g_oLogFile, "0x%08X%08X,", getLogicalSwitchesStates(32), getLogicalSwitchesStates(0));
+
+      for (uint8_t channel = 0; channel < MAX_OUTPUT_CHANNELS; channel++) {
+        f_printf(&g_oLogFile, "%d,", 1500+channelOutputs[channel]/2); // in us
```

Comment:
Please use constants instead of magic numbers (`1500`). I believe this already defined as `PPM_CENTER`.